subprojects/docs/src/docs/userguide/repository_types.adoc

```diff
@@ -235,6 +235,16 @@ The following metadata sources are supported:
 The defaults for Ivy and Maven repositories change with Gradle 5.0. Before 5.0, `artifact()` was included in the defaults. Leading to some inefficiency when modules are missing completely. To restore this behavior, for example, for Maven central you can use `mavenCentral { mavenPom(); artifact() }`. In a similar way, you can opt into the new behavior in older Gradle versions using `mavenCentral { mavenPom() }`
 ====
 
+
+When using `mavenPom` or `ivyDescriptor`, Gradle will look for a marker header to know if a module file exists and look for it, this is known as `RedirectingGradleMetadataModuleMetadataSource`. Starting with 5.6, Gradle also supports disabling this behavior by adding `ignoreGradleMetadataRedirection`.
```

Comment:
Suggested change:
> Since Gradle 5.3, when parsing a metadata file, be it Ivy or Maven, Gradle will look for a marker indicating that a matching Gradle Module Metadata files exists. If it is found, it will be used instead of the Ivy or Maven file. Starting with Gradle 5.6, you can disable this behavior by adding `ignoreGradleMetadataRedirection()` to the `metadataSources` declaration.